docs/en_us/internal/testing.rst

```diff
@@ -281,7 +281,7 @@ console::
     paver test_js_run -s common
     paver test_js_run -s common-requirejs
 
-To run JavaScript tests in a browser:
+To run JavaScript tests in a browser::
```

Comment:
Hello @lamagnifica !
I though this was the common practice, as a lot of the other list headers are of that format. Should I change those as well?